src/VirtualClient/VirtualClient.Actions/MLPerfTraining/MLPerfTrainingExecutor.cs

```diff
@@ -0,0 +1,380 @@
+        {
+            get
+            {
+                return this.Parameters.GetValue<string>(nameof(this.BatchSize), "40");
+            }
+        }
+
+        /// <summary>
+        /// The framework used for the implementation. It will be a sub directory in the model/implementations directory
+        /// </summary>
+        public string Implementation
+        {
+            get
+            {
+                return this.Parameters.GetValue<string>(nameof(this.Implementation), "pytorch-22.09");
+            }
+        }
+
+        /// <summary>
+        /// Container image name
+        /// </summary>
+        public string ContainerName
+        {
+            get
+            {
+                return this.Parameters.GetValue<string>(nameof(this.ContainerName), "language_model");
+            }
+        }
+
+        /// <summary>
+        /// Name of zip file without .zip. It will be the name of the folder where the model package is extracted to
+        /// </summary>
+        public string DataPath
+        {
+            get
+            {
+                return this.Parameters.GetValue<string>(nameof(this.DataPath), "mlperf-training-data-bert.1.0.0");
+            }
+        }
+
+        /// <summary>
+        /// Number of GPUs to be utilized
+        /// </summary>
+        public int GPUCount
+        {
+            get
+            {
+                return this.Parameters.GetValue<int>(nameof(this.GPUCount), 8);
+            }
+        }
+
+        /// <summary>
+        /// Config file for MLPerf Training
+        /// </summary>
+        public string ConfigFile
+        {
+            get
+            {
+                return this.Parameters.GetValue<string>(nameof(this.ConfigFile), "config_DGXA100_1x8x56x1.sh");
+            }
+        }
+
+        /// <summary>
+        /// The NVIDIA implementation for the MLPerf Training Workload, which is a sub-directory in the repository home
+        /// </summary>
+        protected string NvidiaPath
+        {
+            get
+            {
+                return this.PlatformSpecifics.Combine(this.PlatformSpecifics.PackagesDirectory, "mlperf", "NVIDIA");
+            }
+        }
+
+        /// <summary>
+        /// The MLPerf Training Pytorch code directory.
+        /// </summary>
+        protected string ExecutionPath
+        {
+            get
+            {
+                return this.PlatformSpecifics.Combine(this.NvidiaPath, "benchmarks", this.Model, "implementations", this.Implementation);
+            }
+        }
+
+        /// <summary>
+        /// Initializes the environment for execution of the MLPerf Training workload.
+        /// </summary>
+        protected override async Task InitializeAsync(EventContext telemetryContext, CancellationToken cancellationToken)
+        {
+            this.Logger.LogTraceMessage($"{this.TypeName}.InitializationStarted", telemetryContext);
+
+            await this.ThrowIfUnixDistroNotSupportedAsync(cancellationToken)
```

Comment:
We do not throw now to support multiple workloads in single profile. IsSupported function need to be overwritten according to supported platform and architecture.